.github/workflows/ci-core.yaml

```diff
@@ -38,9 +38,13 @@ jobs:
 
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [12, 14]
```

Comment:
I think we probably want to also test against 15, to catch issues before we upgrade, but not critical we do that now.